src/os_posix/os_fs.c

```diff
@@ -612,6 +612,10 @@ __posix_file_write(
               "%s: handle-write: pwrite: failed to write %" WT_SIZET_FMT
               " bytes at offset %" PRIuMAX,
               file_handle->name, chunk, (uintmax_t)offset);
+        if (wt_session->app_private != NULL &&
+          WT_PREFIX_MATCH((char *)wt_session->app_private, "TID"))
+            WT_RET(__wt_msg(session, "pwrite: %s: %s offset %" PRIu64 " len %" PRIu64,
+              file_handle->name, (char *)wt_session->app_private, (uint64_t)offset, (uint64_t)len));
```

Comment:
@sueloverso, why not add the `format-mirror` branches arguments, maybe renamed to `test_{private,private_arg}`, then we don't have to worry about collisions in `app_private`.
```
    void (*format_private)(int, void *); /* Format test program private callback. */
    void *format_private_arg;
```

Reply:
That is reasonable. And I'm going to add it to `random_abort` because although it hasn't hit, it is similar and uses the log in the same way.